### PR TITLE
transfer annual_bias script to diags

### PR DESCRIPTION
**Pull request by raphaeldussin**
@jkrasting preparing the terrain to get the other diags in.
Do see any issues with this PR?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
     license="GPLv3",
     keywords="",
     url="https://github.com/raphaeldussin/om4labs",
-    packages=["om4labs"],
+    packages=["om4labs", "om4labs/diags",
+              "om4labs/diags/annual_bias_1x1deg"],
```
Comment (jkrasting) [inline setup.py:14]: Should we list each script separately, or just rely on the super wrapper?

File: scripts/om4labs [hunk 2]
```diff
@@ -11,21 +11,22 @@ class CLI(object):
...
     def __init__(self):
-        usage = "om4plot <command> [<args>]\n\n"
+        usage = "om4labs <command> [<args>]\n\n"
         for diag in avail_diags:
-            _description = eval(f"om4analysis.diags.{diag}.__description__")
+            _description = eval(f"om4labs.diags.{diag}.__description__")
             usage = usage + (f"    {diag}  {_description}\n")
 
-        parser = argparse.ArgumentParser(
+        metaparser = argparse.ArgumentParser(
```
Comment (jkrasting) [inline scripts/om4labs:22]: Nice change.  Makes this more clear.

File: setup.py (hunk 1 above)
Comment (raphaeldussin) [reply setup.py:14]: for some obscure reason I could not get the diags to load properly without adding explicitly the different subdirectories. maybe an issue to revisit later

**Comment by raphaeldussin** [comment]
> Should we attempt to standardize the names of the read(), compute() and plot() functions across the scripts? These are the names I chose for the MOC and sea ice scripts.

yes definitely! I'd also suggest:

```python

def parse_and_run

def parse

def run

```

I didn't want to make too many changes at once so the PR stays readable but I'll make a new one for this.

**Pull request merged**